From 7b263aed921afdf92525975befd268b0871b8dc4 Mon Sep 17 00:00:00 2001
From: Michael Droettboom <mdboom@gmail.com>
Date: Fri, 18 Mar 2022 19:59:46 -0700
Subject: [PATCH 3/7] lapack install make

lapack_install requires f2clib (see INSTALL/Makefile).

Don't run LAPACK_INSTALL tests since they won't actually run on x86
---
 Makefile | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 36c6a0d..4e43b79 100644
--- a/Makefile
+++ b/Makefile
@@ -15,9 +15,8 @@ lib: f2clib blaslib variants lapacklib tmglib
 
 clean: cleanlib cleantesting cleanblas_testing 
 
-lapack_install:
-	( cd INSTALL; $(MAKE); ./testlsame; ./testslamch; \
-	  ./testdlamch; ./testsecond; ./testdsecnd; ./testversion )
+lapack_install: f2clib
+	( cd INSTALL; $(MAKE) )
 
 blaslib:
 	( cd BLAS/SRC; $(MAKE) )
-- 
2.25.1

